bins/revm-test/src/bin/snailtracer.rs

```diff
@@ -23,21 +23,12 @@ pub fn simple_example() {
     );
     evm.env.tx.data = Bytes::from(hex::decode("30627b7c").unwrap());
 
-    let mut elapsed = std::time::Duration::ZERO;
-    let mut times = Vec::new();
-    for _ in 0..30 {
-        let timer = Instant::now();
+    // Microbenchmark
+    let bench_options = microbench::Options::default().time(Duration::from_secs(3));
+
+    microbench::bench(&bench_options, "Snailtracer benchmark", || {
         let (_, _) = evm.transact();
```

Comment:
Should this be black boxed?

Reply:
Should not matter? it is called for 3s and not N times and for some warmup.